executor/common_linux.h

```diff
@@ -4103,3 +4103,281 @@ static void setup_usb()
 // The macro is used in generated C code.
 #define CAST(f) ({void* p = (void*)f; p; })
 #endif
+
+#if SYZ_EXECUTOR || __NR_syz_mount_fuse
+#include <fcntl.h>
+#include <linux/fuse.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/mount.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+/* Perform correct FUSE initialization:
+ * 1) Create target directory if missing
+ * 2) Mount FUSE fs on top of target directory
+ * 3) Return fd referring to the target directory
+ */
+static volatile long syz_mount_fuse(volatile long a0, /* /dev/fuse fd. */
```

Comment:
Please prefix all FUSE pseudo-syscalls with `syz_fuse_`. E.g. `syz_fuse_mount` instead of `syz_mount_fuse`, etc.

Reply:
Done.